### PR TITLE
fby35: hd: Adjust LTC4282 power and current

### PR DESCRIPTION
**Pull request by Yi-Shum**
Summary:
- Adjust the power and current of the LTC4282.

Test Plan:
- Build code: PASS

### CODE REVIEW

File: meta-facebook/yv35-hd/src/platform/plat_hook.c [hunk 1]
```diff
@@ -129,3 +131,25 @@ bool post_adm1278_pwr_read(uint8_t sensor_num, void *args, int *reading)
 
 	return true;
 }
+
+bool post_ltc4282_cur_read(uint8_t sensor_num, void *args, int *reading)
+{
+	sensor_val *sval = (sensor_val *)reading;
```
Comment (GoldenBug) [inline meta-facebook/yv35-hd/src/platform/plat_hook.c:137]: Check reading to see if it's NULL.

There's a common macro we've added for that which also logs errors.

File: meta-facebook/yv35-hd/src/platform/plat_hook.c [hunk 2]
```diff
@@ -129,3 +131,25 @@ bool post_adm1278_pwr_read(uint8_t sensor_num, void *args, int *reading)
...
+	float val = sval->integer + (sval->fraction * 0.001);
+	val = ADJUST_LTC4282_CURRENT(val);
+	sval->integer = (int16_t)val;
+	sval->fraction = (val - sval->integer) * 1000;
+
+	return true;
+}
+
+bool post_ltc4282_pwr_read(uint8_t sensor_num, void *args, int *reading)
```
Comment (GoldenBug) [inline meta-facebook/yv35-hd/src/platform/plat_hook.c:146]: Check reading to see if it's NULL.

There's a common macro we've added for that which also logs errors.

**Review by GoldenBug — CHANGES_REQUESTED** [review]

File: meta-facebook/yv35-hd/src/platform/plat_hook.c (hunk 1 above)
Comment (Yi-Shum) [reply meta-facebook/yv35-hd/src/platform/plat_hook.c:162]: OK, will use the common macro to check if "reading" is NULL.

File: meta-facebook/yv35-hd/src/platform/plat_hook.c (hunk 2 above)
Comment (Yi-Shum) [reply meta-facebook/yv35-hd/src/platform/plat_hook.c:171]: OK, will use the common macro to check if "reading" is NULL.

**Comment by Yi-Shum** [comment]
Hi @GoldenBug,
I has modify it according to your comment.
Please review it again, thanks.

**Review by GoldenBug — APPROVED** [review]

**Pull request closed without merging**